lib/cubits/statistics/statistics_state.dart

```diff
@@ -33,21 +33,16 @@ extension DropdownName on StatisticsFilterCategory {
 class StatisticsState extends Equatable {
```

Comment:
Should look at refactoring this state class to multiple classes now that it has been simplified